src/OpenTelemetry.Exporter.Jaeger/Implementation/JaegerActivityExtensions.cs

```diff
@@ -128,7 +128,7 @@ public static JaegerSpan ToJaegerSpan(this Activity activity)
                 operationName: activity.DisplayName,
                 flags: (activity.Context.TraceFlags & ActivityTraceFlags.Recorded) > 0 ? 0x1 : 0,
                 startTime: ToEpochMicroseconds(activity.StartTimeUtc),
-                duration: (long)activity.Duration.TotalMilliseconds * 1000,
+                duration: activity.Duration.Ticks * TicksPerMicrosecond,
```

Comment:
Sorry I wasn't paying close attention to this - is this `*` or `/`?

Reply:
Yes, my bad. That's embarrassing 😓